documentation/modules/exploit/linux/http/panos_readsessionvars.md

```diff
@@ -0,0 +1,27 @@
+This module exploits a chain of vulnerabilities in Palo Alto Networks products running
```

Comment:
h00die likes documentation to start with a `## Description` heading, and contain a `## Scenarios` section at the end with example module output.